src/cuda/device_conj.cu

```diff
@@ -0,0 +1,72 @@
+#include "blas/device.hh"
+#include "thrust/complex.h"
+
+#if defined(BLAS_HAVE_CUBLAS)
+
+namespace blas {
+
```

Comment:
This looks good. I find it beneficial to put a regular (non-doxygen) comment before the kernel explaining how the blocks and threads are organized, what each thread is doing. In this case, it's fairly obvious.

Reply:
Cf. magma/magmablas/zsymv.cu and other examples in magmablas.